### ISSUE TITLE
[ui5-li]: accessibilityAttributes API doesn't work as expected

### ISSUE THREAD

**Issue by unazko** [UI5/webcomponents#12467]
### Bug Description

The accessibilityAttributes API of the `ui5-li` doesn't seem to work.

### Affected Component

ui5-li

### Expected Behaviour

When the `accessibilityAttributes` are defined that should result in particular attributes appearing in the list item element.

### Isolated Example

https://stackblitz.com/edit/js-mhnaa2?file=index.js,index.html

### Steps to Reproduce

We have: 
```
accessibilityAttributes = {
    hasPopup: "menu"
}
```
for both the `ui5-button` and the `ui5-li` items.

Observe that the `ui5-button` has `aria-haspopup=menu` attribute while the `ui5-li` elements do not have this attribute.

### Priority

Medium

### UI5 Web Components Version

latest

### Browser

Chrome

### Operating System

macOS

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by unazko** [comment]
Hello @UI5/ui5-team-ctr-pir,

Could you please check if this is an issue with how the `accessibilityAttributes` API should work.
If the `accessibilityAttributes` value is provided in the browser debugger then it works fine.
It doesn't work in the sample however: https://stackblitz.com/edit/js-mhnaa2?file=index.js,index.html
We've identified this behaviour recently while checking if we can add a `ui5-menu` to be opened as a context menu next to a `ui5-li` item. In that case we could set the `aria-haspopup` attribute.

Best regards,
Boyna

**Comment by dobrinyonkov** [comment]
Hi @unazko, 

This issues seems to no longer reproduce, the aria-haspopup attribute is correctly set. Feel free to re-open if you still face issues with the API.

Kind Regards,
Dobrin
